### PR TITLE
Upgrade instructions KF 1.2 to 1.3

### PR DESCRIPTION
**Pull request by jppgks**
Still very raw - happy to address comments and requested changes.

### CODE REVIEW

**Comment by animeshsingh** [comment]
thanks @jppgks 
cc @Tomcli @moficodes

File: HandsOn/upgrade/KF_upgrade_1_2_to_1_3.md [hunk 1]
```diff
@@ -0,0 +1,321 @@
+# Kubeflow upgrade 1.2 to 1.3 (multi-user, AppID, IKS, K8s v1.19)
+_Raw Technical Writeup_
+
+*Credits to Tommy Li and Mofi Rahman!*
+
+Prerequisites:
+- bash
+- kubectl, logged in to cluster
+- old KF 1.2 installation
```
Comment (Tomcli) [inline HandsOn/upgrade/KF_upgrade_1_2_to_1_3.md]: FYI, 1.2 single user have different database schema for the runs and experiments. Although the upgrade process will be similar, we need to apply the namespace column from #5 to all the db tables.
```suggestion
- old KF 1.2 multi-user installation
```

**Comment by Tomcli** [comment]
thanks @jppgks, it looks good to me overall

**Comment by Tomcli** [comment]
thanks @jppgks for your contributions.

**Comment by habmat** [comment]
Looks good

**Pull request merged**